src/media/class-wpml-page-builders-media-translate.php

```diff
@@ -51,6 +51,10 @@ public function translate_image_url( $url, $lang, $source_lang ) {
 	 * @return int
 	 */
 	public function translate_id( $id, $lang ) {
+		if ( (int) $id < 1 ) {
```

Comment:
@andreasciamanna, the purpose of this change is to make sure we have a valid ID. I fear that we could have some cases with non-numeric strings in the future (when the method will receive arguments coming from shortcode configuration). This will prevent fatal errors.